### PR TITLE
Dynamic data for progressbar

### PR DESCRIPTION
**Pull request by tguelenman**
not sure where to merge into
@adrianpfeiffer FYI
- all the data is now read correctly
- i also implemented today's new cases in germany, as you wanted
- i didnt get to do the circular % stuff, if you really want this, please give it a try yourself

### CODE REVIEW

File: lib/overlays/menu_overlay.dart [hunk 1]
```diff
@@ -73,7 +75,11 @@ class MenuOverlay extends StatelessWidget {
                 }),
             // PrivacyPolicyButton
             Expanded(
-              flex: 2,
+              flex: 1,
+              child: ProgressBar(),
```
Comment (abegehr) [inline lib/overlays/menu_overlay.dart:79]: should the progressbar be on the Menuscreen?
@adrianpfeiffer, @tguelenman

File: lib/utils/score.dart [hunk 2]
```diff
@@ -31,8 +32,27 @@ class Score {
     return uuid;
   }
 
+  static incrementGamesCounter() async {
```
Comment (abegehr) [inline lib/utils/score.dart:35]: games should also be counted on mobile.

File: lib/components/progress_bar.dart [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+import 'package:cloud_firestore/cloud_firestore.dart';
+
+class ProgressBar extends StatefulWidget {
+  @override
+  ProgressBarState createState() => ProgressBarState();
+}
+
+class ProgressBarState extends State<ProgressBar> {
+  Future<String> fetchData() async {
+    final response = await http.get('https://corona.lmao.ninja/countries');
```
Comment (abegehr) [inline lib/components/progress_bar.dart:15]: You can directly fetch data for Germany: https://corona.lmao.ninja/docs/#/Countries/get_countries__country_country_____

File: lib/components/progress_bar.dart [hunk 4]
```diff
@@ -0,0 +1,176 @@
...
+
+    if (response.statusCode == 200) {
+      final parsedResponse =
+          jsonDecode(response.body).cast<Map<String, dynamic>>();
+
+      final list = parsedResponse
+          .map<Country>((json) => Country.fromJson(json))
```
Comment (abegehr) [inline lib/components/progress_bar.dart:22]: this might be overkill for getting data for one country

File: lib/overlays/menu_overlay.dart (hunk 1 above)
Comment (tguelenman) [reply lib/overlays/menu_overlay.dart:79]: Yes

File: lib/utils/score.dart (hunk 2 above)
Comment (tguelenman) [reply lib/utils/score.dart:35]: @abegehr what makes you think they are not?

File: lib/components/progress_bar.dart (hunk 3 above)
Comment (tguelenman) [reply lib/components/progress_bar.dart:15]: that would be better indeed, but i dont want to spend more time on this. can we leave it as is?

File: lib/components/progress_bar.dart (hunk 4 above)
Comment (tguelenman) [reply lib/components/progress_bar.dart:22]: that would be better indeed, but i dont want to spend more time on this. can we leave it as is?

File: lib/utils/score.dart (hunk 2 above)
Comment (abegehr) [reply lib/utils/score.dart:35]: this is merged into the coronajump branch only

File: lib/components/progress_bar.dart (hunk 3 above)
Comment (abegehr) [reply lib/components/progress_bar.dart:15]: ok

File: lib/components/progress_bar.dart (hunk 4 above)
Comment (abegehr) [reply lib/components/progress_bar.dart:22]: ok

**Comment by abegehr** [comment]
@tguelenman please re-request reviews, when needed

**Comment by tguelenman** [comment]
@abegehr this is not a matter of attention, but of not knowing. If this does not belong in coronajump or master then where does it belong? what makes you write such a comment

> The progressbar should not be merged into master. @tguelenman, please pay attention.

instead of being helpful and enlightening me on where it belong? Honest question - this feels like a scheme of communication that is happening too often and it's bothering me, so I'd like to try and understand your point of view better

**Pull request merged**